OnPod/Node/Examples/ADC/adc.cpp

```diff
@@ -0,0 +1,47 @@
+#include <Arduino.h>
+#include <SPI.h>
+#include "adc.h"
+
+ExtADC::ExtADC(uint16_t iActiveChannels) : iActiveChannels(iActiveChannels) {
```

Comment:
Traditionally the name of a SPI driver reflects the model number/device class of the device the driver interacts with: In this case an ADS7953. I'd recommend naming this as such with the initializer taking the appropriate chip select pin and other SPI specific settings (but not initializing the SPI bus in the constructor). This will also make it easier when the node board is rev'd in ways that change pin mappings.

Also, having enableChannel/disableChannel helpers would be much cleaner IMO than passing a 16 bit bitmask to a constructor.